apps/rush-lib/src/logic/PackageChangeAnalyzer.ts

```diff
@@ -29,9 +31,12 @@ export class PackageChangeAnalyzer {
     this._git = new Git(this._rushConfiguration);
   }
 
-  public getPackageDeps(projectName: string): Map<string, string> | undefined {
+  public async getPackageDeps(
```

Comment:
This code uses a suffix to indicate async methods, so the name should probably be changed to `getPackageDepsAsync()`.  The same applies to other functions that were converted to async in this PR.